src/cuda/common.cu

```diff
@@ -104,97 +170,39 @@ void set_nf_type12(CUFINUFFT_BIGINT ms, cufinufft_opts opts, finufft_spread_opts
   }
 }
 
-template<typename T>
-void onedim_fseries_kernel(CUFINUFFT_BIGINT nf, T *fwkerhalf, finufft_spread_opts opts)
-/*
-  Approximates exact Fourier series coeffs of cnufftspread's real symmetric
-  kernel, directly via q-node quadrature on Euler-Fourier formula, exploiting
-  narrowness of kernel. Uses phase winding for cheap eval on the regular freq
-  grid. Note that this is also the Fourier transform of the non-periodized
-  kernel. The FT definition is f(k) = int e^{-ikx} f(x) dx. The output has an
-  overall prefactor of 1/h, which is needed anyway for the correction, and
-  arises because the quadrature weights are scaled for grid units not x units.
-
-  Inputs:
-  nf - size of 1d uniform spread grid, must be even.
-  opts - spreading opts object, needed to eval kernel (must be already set up)
-
-  Outputs:
-  fwkerhalf - real Fourier series coeffs from indices 0 to nf/2 inclusive,
-              divided by h = 2pi/n.
-              (should be allocated for at least nf/2+1 Ts)
-
-  Compare onedim_dct_kernel which has same interface, but computes DFT of
-  sampled kernel, not quite the same object.
-
-  Barnett 2/7/17. openmp (since slow vs fftw in 1D large-N case) 3/3/18
-  Melody 2/20/22 separate into precomp & comp functions defined below.
- */
-{
-  T f[MAX_NQUAD];
-  std::complex<double> a[MAX_NQUAD];
-  onedim_fseries_kernel_precomp(nf, f, a, opts);
-  onedim_fseries_kernel_compute(nf, f, a, fwkerhalf, opts);
-}
-
 /*
   Precomputation of approximations of exact Fourier series coeffs of cnufftspread's
   real symmetric kernel.
 
   Inputs:
   nf - size of 1d uniform spread grid, must be even.
   opts - spreading opts object, needed to eval kernel (must be already set up)
+  phase_winding - if true, compute normalization factors for phase winding rates,
+                  otherwise compute scaled quadrature nodes
 
   Outputs:
-  a - phase winding rates
+  a - normalization factors if phase winding is true, otherwise scaled quadrature nodes
```

Comment:
really, a is always a vector of scaled quadrature nodes; the scaling happens to be different for type 1,2 (equispaced) vs type 3 (general kx,ky,kz).  Really having them both called "a" is confusing... see previous review.